src/backend/looped/vehicle/disable_siren.cpp

```diff
@@ -0,0 +1,29 @@
+#include "backend/looped_command.hpp"
+#include "natives.hpp"
+
+namespace big
+{
+	class siren_mute : looped_command
+	{
+		using looped_command::looped_command;
+
+		virtual void on_tick() override
+		{
+			if (ENTITY::DOES_ENTITY_EXIST(self::veh))
+				{
```

Comment:
This tab after `if` is weird.

Reply:
There's some real issue with this Git UI, There's not even a Tab there in my local repo. I'll try fix that

Reply:
I tried lemme know if its still there in ur pov as in my pov it just changed (like it was before).

Reply:
Seems fixed for me.

Reply:
Okay, Thanks.